core/src/main/java/org/apache/calcite/rex/RexSimplify.java

```diff
@@ -557,6 +561,29 @@ private RexNode simplifyIsNull(RexNode a) {
     }
   }
 
+  /**
+   * Simplifies "f(x0,x1,...)" to "null" only if f's strong policy is ANY
```

Comment:
Frankly speaking, `strong` is in the function name. It does not clarify things.

It would be so much better if you added an example of such a function

Reply:
done! please take a look,thanks!